.github/workflows/psalm.yml

```diff
@@ -57,3 +57,8 @@ jobs:
 
       - name: "psalm"
         run: "vendor/bin/psalm --shepherd --stats"
+        env:
+            GIT_AUTHOR_NAME: "Laminas Automatic Releases"
```

Comment:
Eh, no, let's initialize repositories in tests, rather than requiring environment to be set

Reply:
The error was from running `git init` within a unit test setup, due to missing author/committer in the env...

Reply:
Aware: the other tests configure a user/email 👍